### DISCUSSION TITLE
Browse PDS

### DISCUSSION THREAD

**Discussion by tom-sherman** [likeandscribe/frontpage#185] [Ideas]
Similar to https://pdsls.pages.dev, when we encounter a PDS URL we should let you paginate through the DIDs from `com.atproto.repo.describeRepo`

As there's no way to programatically detect a PDS given simply a URL, we'd probably need to add some special case logic to the DID doc UI similar to how we special case blob objects.

**Comment by tom-sherman** [comment]
Checking for an OK JSON response from `com.atproto.server.describeServer` could work for detection actually.

**Reply by syui** [reply]
I have encountered a similar problem before. Since there is no API to get the connected PLC, I gave up on the idea of ​​solving the problem fundamentally.

I think I once asked bsky.team on github "Is there something that will return the PLC that pds is using?"

[google translate]
